packages/blocks/countdown/src/base.css

```diff
@@ -0,0 +1,258 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700;800&display=swap");
+
+/* –––––––––– Root variables –––––––––– */
+
+:root {
```

Comment:
This CSS is incompatible with un-sandboxed blocks. Not sure what to replace this CSS with if we want optional sandboxing. @CiaranMn any thoughts?

Reply:
We spoke about this – in this case it looks like the variables are mostly un-used, so I would move any that are actually used to `styles.css`, where there are already some scoped, and delete the file.

We could additionally make `styles.css` a CSS module (renaming it to `styles.module.css` might work automatically, unsure) to ensure no clashes with other classes called `display-grid`.

Reply:
I scoped the styles to the block's class name for now in [5d5eecb](https://github.com/hashintel/hash/pull/583/commits/5d5eecb73e9332c071fc32b269b9e35ee3b68eda). Couldn't find a quick fix to the bug I mentioned earlier about the imported styles object being empty. I believe that will be address in [internal task](https://app.asana.com/0/1201095311341924/1202361526477995)

Reply:
We can just delete this then, right? It's still adding global styles that it shouldn't

Reply:
Deleted in [78a4bfa](https://github.com/hashintel/hash/pull/583/commits/78a4bfacddf8a51534e8c32c843440b435045144)